apps/agent/src/mastra/tools/github.ts

```diff
@@ -64,25 +69,40 @@ export const getPullRequestTool = createTool({
     })
 
     return {
-      list: prs.map((pr) => ({
-        number: pr.number,
-        title: pr.title,
-        url: pr.html_url,
-        author: pr.user.login,
-        createdAt: pr.created_at,
-        updatedAt: pr.updated_at,
-        mergedAt: pr.merged_at,
-        isMerged: pr.merged_at !== null,
-        hasMergeConflicts: githubClient.hasMergeConflicts(pr),
-        draft: pr.draft,
-        isWIP: githubClient.isWIP(pr),
-        isWaitingForReview: githubClient.isWaitingForReview(pr),
-        labels: pr.labels.map((label) => label.name),
-        reviewers: pr.requested_reviewers.map((reviewer) => reviewer.login),
-        hasComments: pr.comments > 0 || pr.review_comments > 0,
-        hasReviews: pr.reviews && pr.reviews.length > 0,
-        body: pr.body,
-      })),
+      list: prs.map((pr) => {
+        let author = pr.user.login
+        let reviewers = pr.requested_reviewers.map((reviewer) => reviewer.login)
+
+        if (context.useDiscordIdMapping) {
+          author =
+            `<@!${githubIdMapper.getDiscordID(author)?.toString()}>` || author
```

Comment:
I think we shouldn't change it into a string. If no record is found, the function will return `undefined`. Using toString() will turn it into the string `undefined`, so the fallback won't work.